src/schema/artwork/index.js

```diff
@@ -450,14 +454,15 @@ export const artworkFields = () => {
     is_price_range: {
       type: GraphQLBoolean,
       resolve: ({ price, edition_sets }) =>
-        has_price_range(price) && !has_multiple_editions(edition_sets), // eslint-disable-line max-len
+        has_price_range(price) &&
+        (has_single_edition(edition_sets) || !has_editions(edition_sets)), // eslint-disable-line max-len
```

Comment:
This works but I think it might have been a bit more readable if we kept `has_multiple_editions` and made it check for more than 1 edition.

Something like:
```
const has_editions = edition_sets => {
   return edition_sets && edition_sets.length > 0
 }

const has_single_edition = edition_sets => {
  return has_editions && edition_sets.length > 1
}
```

And then here the code would not change at all and the change that you did below still valid. Just another way to do the same thing. I personally find combination of `&&`, `||` and `!` pretty confusing to read.

Reply:
Oh! Nice, yeah, good catch.